src/Nethermind/Nethermind.Merge.Plugin/MergeConfig.cs

```diff
@@ -22,6 +22,7 @@ namespace Nethermind.Merge.Plugin
 {
     public class MergeConfig : IMergeConfig
     {
+        // No longer needed, but we can't remove it because backward compatibility. This settings is ignored
```

Comment:
What would happen if we remove it?

Reply:
Let's remove from doc's as minimum?

Reply:
try running node with --Merge.TestEnabled true for example. You will get
2022-10-13 13:37:57.8521|Failure Microsoft.Extensions.CommandLineUtils.CommandParsingException: Unrecognized option '--Merge.TestEnabled'
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.HandleUnexpectedArg(CommandLineApplication command, String[] args, Int32 index, String argTypeName)
   at Microsoft.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at Nethermind.Runner.Program.Run(String[] args) in C:\Users\marek\Desktop\Nethermind\nethermind\nethermind\src\Nethermind\Nethermind.Runner\Program.cs:line 228
   at Nethermind.Runner.Program.Main(String[] args) in C:\Users\marek\Desktop\Nethermind\nethermind\nethermind\src\Nethermind\Nethermind.Runner\Program.cs:line 85


What is more, we can have one case when we don't have TTD and we want to have Merge.Enabled => bellatrix on CL

Reply:
cool I am conviced.